ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor.hs

```diff
@@ -100,13 +102,41 @@ inboundGovernor tracer serverControlChannel protocolIdleTimeout
           }
     inboundGovernorLoop state
   where
+    inboundGovernorStatetoCounters
+      :: InboundGovernorState muxMode peerAddr m a b
+         -> m InboundGovernorCounters
+    inboundGovernorStatetoCounters InboundGovernorState { igsConnections } = do
+      chainSyncStatus <- atomically (getChainSyncStatus igsConnections)
+
+      let hotPeersRemote = length $ filter (== StatusRunning) chainSyncStatus
+
+      return $ InboundGovernorCounters { hotPeersRemote }
+      where
+        -- Due to circular dependencies it is not possible to import
+        -- Ouroboros.Network.NodeToNode (chainSyncMiniProtocolNum)
+        chainSyncMiniProtocolNum :: MiniProtocolNum
+        chainSyncMiniProtocolNum = MiniProtocolNum 2
```

Comment:
Could you break out the protocol numbers into its own module, for example Ouroboros.Network.NodeToNode.MiniProtocolNums . ?

Reply:
The problem is that protocol numbers are defined at `ouroboros-network` repo and `ouroboros-network` depends on `ouroboros-network-framework` so I can not import that module